src/FolioServiceProvider.php

```diff
@@ -18,6 +18,7 @@ public function register(): void
         $this->app->singleton(FolioManager::class);
         $this->app->singleton(InlineMetadataInterceptor::class);
         $this->app->singleton(FolioRoutes::class);
+        $this->app->bind(Router::class);
```

Comment:
I'm not sure this is even necessary? This doesn't really do anything.

Reply:
@taylorotwell Thank you for taking the time to review and add this comment, in addition to improving and merging the PR.